dtmi/azure/iot/deviceupdatecontractmodel-2.json

```diff
@@ -0,0 +1,15 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azure:iot:deviceUpdateContractModel;2",
+  "@type": "Interface",
+  "displayName": "Device Update Base Model version 2",
+  "description": "Device Update Base Model version 2 supports device update interface version 1 and update manifest version 5.",
```

Comment:
I think the description should be use to inform users of this interface capabilties, rather than describe the internal versioning update.. 

You can always add these comments in the PR itself :)

same for the next description